src/status_im/contexts/communities/channel_details/view.cljs

```diff
@@ -0,0 +1,183 @@
+        {:keys [admins] :as group} (rf/sub [:chats/chat-by-id chat-id])
+        admin?                     (get admins (rf/sub [:multiaccount/public-key]))]
+    [rn/view {:flex 1 :margin-top 20}
+     [rn/touchable-opacity
+      {:on-press            #(rf/dispatch [:navigate-back])
+       :accessibility-label :close-manage-members
+       :style               (style/close-icon)}
+      [quo/icon :i/close {:color (colors/theme-colors colors/neutral-100 colors/white)}]]
+     [quo/text
+      {:size   :heading-1
+       :weight :semi-bold
+       :style  {:margin-left 20}}
+      (i18n/label (if admin? :t/manage-members :t/add-members))]
+     [rn/section-list
+      {:key-fn                         :title
+       :sticky-section-headers-enabled false
+       :sections                       (rf/sub [:contacts/grouped-by-first-letter])
+       :render-section-header-fn       contact-list/contacts-section-header
+       :content-container-style        {:padding-bottom 20}
+       :render-data                    {:group group}
+       :render-fn                      add-member-contact-item-render}]
+     [rn/view {:style (style/bottom-container 30)}
+      [quo/button
+       {:container-style     {:flex 1}
+        :type                :primary
+        :accessibility-label :save
+        :on-press            (fn []
+                               (rf/dispatch [:navigate-back])
+                               (js/setTimeout (fn []
+                                                (rf/dispatch
+                                                 [:group-chats.ui/remove-members-pressed chat-id]))
+                                              500)
+                               (rf/dispatch [:group-chats.ui/add-members-pressed chat-id]))
+        :disabled?           (and (zero? (count selected-participants))
+                                  (zero? (count deselected-members)))}
+       (i18n/label :t/save)]]]))
+
+(defn contact-item-render
+  [{:keys [public-key] :as item} _ _ extra-data]
+  (let [current-pk           (rf/sub [:multiaccount/public-key])
+        show-profile-actions #(rf/dispatch [:show-bottom-sheet
+                                            {:content (fn [] [actions/contact-actions item
+                                                              extra-data])}])]
+    [contact-list-item/contact-list-item
+     (when (not= public-key current-pk)
+       {:on-press                #(rf/dispatch [:chat.ui/show-profile public-key])
+        :allow-multiple-presses? true
+        :on-long-press           show-profile-actions
+        :accessory               {:type     :options
+                                  :on-press show-profile-actions}})
+     item]))
+
+(defn contacts-section-header
+  [{:keys [title]}]
+  [quo/divider-label {:tight? true} title])
+
+(defn contacts-section-footer
+  [_]
+  [rn/view {:style {:height 8}}])
+
+(defn actions
+  [chat-id cover-bg-color]
```

Comment:
why cover-bg-color and not just `chat-color`? it's the custom color of the given chat no? 🤔